### ISSUE TITLE
ubuntu14/16/debian8: fail to install scylla-tools-core in upgrading scylla

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#2732]
*Installation details*
Scylla version (or git commit hash): 2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty
Cluster size:
OS (RHEL/CentOS/Ubuntu/AWS AMI):

*Description*
Upgrade from 1.7.4-0.20170726.ff643e3-0ubuntu1 to 2.0~rc2-0.20170815.8370e1b-0ubuntu1
scylla-tools-core can't installed by `sudo apt-get dist-upgrade scylla`
But we can install it by `sudo apt-get install scylla-tools-core` after dist-upgrade command, then it will be installed successfully. And scylla works well.

Rollback works well.

```
scylla-test@amos-ubuntu14:~$ sudo apt-get dist-upgrade scylla
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following package was automatically installed and is no longer required:
  collectd
Use 'apt-get autoremove' to remove it.
The following NEW packages will be installed:
  num-utils python3-pyudev scylla-tools-core
The following packages will be upgraded:
  scylla scylla-conf scylla-jmx scylla-kernel-conf scylla-server scylla-tools
6 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
Need to get 38.6 MB of archives.
After this operation, 5,092 kB of additional disk space will be used.
Do you want to continue? [Y/n]
WARNING: The following packages cannot be authenticated!
  scylla-conf scylla-server scylla-jmx scylla-tools-core scylla-tools
  scylla-kernel-conf scylla
Install these packages without verification? [y/N] y

Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
The following package was automatically installed and is no longer required:
  collectd
Use 'apt-get autoremove' to remove it.
The following NEW packages will be installed:
  num-utils python3-pyudev scylla-tools-core
The following packages will be upgraded:
  scylla scylla-conf scylla-jmx scylla-kernel-conf scylla-server scylla-tools
6 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
Need to get 38.6 MB of archives.
After this operation, 5,092 kB of additional disk space will be used.
Do you want to continue? [Y/n]
WARNING: The following packages cannot be authenticated!
  scylla-conf scylla-server scylla-jmx scylla-tools-core scylla-tools
  scylla-kernel-conf scylla
Install these packages without verification? [y/N] y

(Reading database ... 44911 files and directories currently installed.)
Preparing to unpack .../num-utils_0.5-11_all.deb ...
Unpacking num-utils (0.5-11) ...
Selecting previously unselected package python3-pyudev.
Preparing to unpack .../python3-pyudev_0.16.1-2build1_all.deb ...
Unpacking python3-pyudev (0.16.1-2build1) ...
Preparing to unpack .../scylla-conf_2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty_amd64.deb ...
Unpacking scylla-conf (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) over (1.7.4-0.20170726.ff643e3-0ubuntu1) ...
Preparing to unpack .../scylla-server_2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty_amd64.deb ...
Unpacking scylla-server (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) over (1.7.4-0.20170726.ff643e3-0ubuntu1) ...
Preparing to unpack .../scylla-jmx_2.0~rc2-20170815.fdfca8e-0ubuntu1~trusty_all.deb ...
Unpacking scylla-jmx (2.0~rc2-20170815.fdfca8e-0ubuntu1~trusty) over (1.7.4-20170726.51ba6ea-0ubuntu1) ...
Selecting previously unselected package scylla-tools-core.
Preparing to unpack .../scylla-tools-core_2.0~rc2-20170815.e77c128-0ubuntu1~trusty_all.deb ...
Unpacking scylla-tools-core (2.0~rc2-20170815.e77c128-0ubuntu1~trusty) ...
dpkg: error processing archive /var/cache/apt/archives/scylla-tools-core_2.0~rc2-20170815.e77c128-0ubuntu1~trusty_all.deb (--unpack):
 trying to overwrite '/usr/share/doc/scylla/cassandra/licenses/joda-time-2.4.txt.gz', which is also in package scylla-tools 1.7.4-20170726.7a15b00-0ubuntu1
dpkg-deb: error: subprocess paste was killed by signal (Broken pipe)
Preparing to unpack .../scylla-tools_2.0~rc2-20170815.e77c128-0ubuntu1~trusty_all.deb ...
Unpacking scylla-tools (2.0~rc2-20170815.e77c128-0ubuntu1~trusty) over (1.7.4-20170726.7a15b00-0ubuntu1) ...
Preparing to unpack .../scylla-kernel-conf_2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty_amd64.deb ...
Unpacking scylla-kernel-conf (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) over (1.7.4-0.20170726.ff643e3-0ubuntu1) ...
Preparing to unpack .../scylla_2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty_amd64.deb ...
Unpacking scylla (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) over (1.7.4-0.20170726.ff643e3-0ubuntu1) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Processing triggers for ureadahead (0.100.0-16) ...
Errors were encountered while processing:
 /var/cache/apt/archives/scylla-tools-core_2.0~rc2-20170815.e77c128-0ubuntu1~trusty_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
scylla-test@amos-ubuntu14:~$ echo $?
100
```

```
scylla-test@amos-ubuntu14:~$ sudo apt-get install scylla-tools-core
Reading package lists... Done  
Building dependency tree
Reading state information... Done
The following package was automatically installed and is no longer required:
  collectd
Use 'apt-get autoremove' to remove it.
The following NEW packages will be installed:
  scylla-tools-core
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
8 not fully installed or removed.
Need to get 0 B/16.2 MB of archives.
After this operation, 18.1 MB of additional disk space will be used.
WARNING: The following packages cannot be authenticated!
  scylla-tools-core
Install these packages without verification? [y/N] y
(Reading database ... 44873 files and directories currently installed.)
Preparing to unpack .../scylla-tools-core_2.0~rc2-20170815.e77c128-0ubuntu1~trusty_all.deb ...
Unpacking scylla-tools-core (2.0~rc2-20170815.e77c128-0ubuntu1~trusty) ...
Setting up num-utils (0.5-11) ...
Setting up python3-pyudev (0.16.1-2build1) ...
Setting up scylla-conf (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) ...
Configuration file '/etc/scylla/scylla.yaml'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** scylla.yaml (Y/I/N/O/D/Z) [default=N] ?
Setting up scylla-server (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) ...
scylla:x:108:114::/var/lib/scylla:/bin/false
scylla:x:114:
Setting up scylla-jmx (2.0~rc2-20170815.fdfca8e-0ubuntu1~trusty) ...
Setting up scylla-tools-core (2.0~rc2-20170815.e77c128-0ubuntu1~trusty) ...
Setting up scylla-tools (2.0~rc2-20170815.e77c128-0ubuntu1~trusty) ...
Setting up scylla-kernel-conf (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) ...
kernel.sched_tunable_scaling = 0
kernel.sched_min_granularity_ns = 500000
kernel.sched_wakeup_granularity_ns = 450000
kernel.sched_latency_ns = 1000000
kernel.sched_autogroup_enabled = 0
kernel.numa_balancing = 0
Setting up scylla (2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty) ...

scylla-test@amos-ubuntu14:~$ dpkg -l|grep scylla
ii  scylla                               2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty amd64        Scylla database metapackage
ii  scylla-conf                          2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty amd64        Scylla database main configuration file
ii  scylla-jmx                           2.0~rc2-20170815.fdfca8e-0ubuntu1~trusty   all          Scylla JMX server binaries
ii  scylla-kernel-conf                   2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty amd64        Scylla kernel tuning configuration
ii  scylla-server                        2.0~rc2-0.20170815.8370e1b-0ubuntu1~trusty amd64        Scylla database server binaries
ii  scylla-tools                         2.0~rc2-20170815.e77c128-0ubuntu1~trusty   all          Scylla database tools
ii  scylla-tools-core                    2.0~rc2-20170815.e77c128-0ubuntu1~trusty   all          Scylla database tools core files
```

@syuu1228

**Comment by tzach** [comment]
@syuu1228 ?

**Comment by tzach** [comment]
@syuu1228  any update?

**Comment by slivne** [comment]
fixed by scylladb/scylla-tools-java@f786892372001ffa769d7a6dcddeb50c774899ae

**Comment by amoskong** [comment]
Verified that the issue had been fixed in 2.0.rc5-0.20170917.0b49cfc.

I will remove workaround in upgrade guide.

**Comment by slivne** [comment]
thanks
